mgmtsystem_action/security/ir.model.access.csv

```diff
@@ -1,5 +1,6 @@
 "id","name","model_id:id","group_id:id","perm_read","perm_write","perm_create","perm_unlink"
 "access_mgmtsystem_action","mgmtsystem.action.user","model_mgmtsystem_action","mgmtsystem.group_mgmtsystem_user",1,1,1,0
+"access_mgmtsystem_claim","mgmtsystem.action.user","crm_claim.model_crm_claim_stage","mgmtsystem.group_mgmtsystem_user",1,0,0,0

```

Comment:
OK for you addition then. 
But it's something that should be fixed upstream: Actions should not be requiring CRM to be installed...